client/e2e/pages/EventsPage.ts

```diff
@@ -232,65 +290,83 @@ export class EventsPage {
     await this.page.getByRole('button', {name: eventFullName}).first().click();
   }
 
+  
   /**
    * We need to pass in a event's full name like:
    * "Test_event Playbill Test_event Description An event for testing"

```

Comment:
You can generate this string within this function instead of relying on adding a full name attribute to the event data object

Reply:
this will reduce typo problems/user error in the future when others create new EVENT_INFO objects